### PR TITLE
chore(release): 2.1.0

### LINKED ISSUE #347: chore(dist): remove dist from version control
Remove dist from version control.

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #347

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+### Features
+
+* **checkbox:** makes the checkboxes accessible ([f0990df](https://github.com/zalando/dress-code/commit/f0990df))
+* **grid:** Add utility classes to hide/show at different breakpoint sizes ([a5a875b](https://github.com/zalando/dress-code/commit/a5a875b)), closes [#287](https://github.com/zalando/dress-code/issues/287)
+* **grid:** Improve functionality and demo ([787c292](https://github.com/zalando/dress-code/commit/787c292))
+* **icons:** Add bar-chart, line-chart, pointer icons ([#356](https://github.com/zalando/dress-code/issues/356)) ([e863f1a](https://github.com/zalando/dress-code/commit/e863f1a))
+* **messages:** Update design of messages. ([2055e43](https://github.com/zalando/dress-code/commit/2055e43)), closes [#329](https://github.com/zalando/dress-code/issues/329)
+* **radio:** makes the radio button accessible ([b17b582](https://github.com/zalando/dress-code/commit/b17b582))
+* **tab:** add tab atom to the dress-code ([03f5711](https://github.com/zalando/dress-code/commit/03f5711))
+* **tab:** add tab atom to the dress-code ([0549bf5](https://github.com/zalando/dress-code/commit/0549bf5))
```
Comment (rbarilani) [inline CHANGELOG.md]: @dami-gg Here you have a duplicated message

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+* **tables:** Make table responsiveness optional ([#330](https://github.com/zalando/dress-code/issues/330)) ([ba80a3a](https://github.com/zalando/dress-code/commit/ba80a3a)), closes [#222](https://github.com/zalando/dress-code/issues/222)
+* **textarea:** add styles for error state ([d1cde40](https://github.com/zalando/dress-code/commit/d1cde40))
+
+
+### BREAKING CHANGES
+
+* breakpoints: * $dc-large-width breakpoint variable renamed to $dc-medium-width
```
Comment (rbarilani) [inline CHANGELOG.md]: @dami-gg Can we fix the formatting here and make the breaking change really visible?

File: CHANGELOG.md (hunk 1 above)
Comment (dami-gg) [inline CHANGELOG.md]: it's not actually duplicated but two different commits with the same message... Shall I remove one anyway?

File: CHANGELOG.md (hunk 2 above)
Comment (dami-gg) [inline CHANGELOG.md]: I'll try to tune it a little bit

File: CHANGELOG.md (hunk 1 above)
Comment (rbarilani) [inline CHANGELOG.md]: @dami-gg merge it one message and leave the two commit links IHMO

**Comment by rbarilani** [comment]
@dami-gg let's also write an email to "tech" mail group when released.

**Comment by dami-gg** [comment]
I need this PR's approval before publishing it @rbarilani

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**